build.gradle

```
@@ -583,7 +583,7 @@ def createCheckTypeTask(projectName, taskName, checker, args = []) {
         outputs.upToDateWhen { false }
         source = project("${projectName}").sourceSets.main.java
         classpath = files(project("${projectName}").compileJava.classpath,project(':checker-qual').sourceSets.main.output)
-        destinationDirectory = file("${buildDir}")
+        destinationDirectory = file("${layout.buildDirectory.get().asFile}")
```

```suggestion
        destinationDirectory = layout.buildDirectory.get().asFile
```

Wouldn't that already be a file?